### DISCUSSION TITLE
Portfolios

### DISCUSSION THREAD

**Discussion by panyiadamz** [community/community#31763] [Other Feature Feedback, Questions, & Ideas]
Can someone help me out? I want to create a portfolio but am finding that hard to do so. How do I go about it?

**Comment by radar-blip** [comment]
You will need to [Create a Repo](https://docs.github.com/en/get-started/quickstart/create-a-repo) and go from there.

**Comment by Kilozen** [comment]
A portfolio of what kind of stuff and what format?  


If you want it to be integrated with/hosted on GitHub have you tried GitHub **Pages**? 
- https://pages.github.com/

**Comment by jfullstackdev** [comment]
you mean the special repository ? create a repo that is exactly the same as your username, edit the README and it will appear as your landing page when someone visited your profile

you must have a repo named `panyiadamz`

now I see that you are really doing the other one, with the GitHub Page, that's fine too, but you need to change your existing repo name to

`panyiadamz.github.io`

make a profile site and that will be available as your profile when someone visited that page
